src/docfx/build/context/LinkResolver.cs

```diff
@@ -54,9 +54,9 @@ internal class DependencyResolver
             return (error, link, file);
         }
 
-        public (Error error, string link, Document file) ResolveLink(SourceInfo<string> path, Document declaringFile)
+        public (Error error, string link, Document file) ResolveAbsoluteLink(SourceInfo<string> path, Document declaringFile)
```

Comment:
Does `ResolveAbsoluteLink` mean the resolved link would be absolute? same for `ResovleAbsoluteXref`

Reply:
yep, to pair with `ResolveRelativeLink`